src/interfaces/alteration-interfaces/business-snapshot-interface.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  BusinessContactIF,
+  BusinessInformationIF,
+  IncorporationAddressIf,
+  NameTranslationSnapshotIF,
+  OrgPersonIF,
+  ShareStructureIF
+} from '@/interfaces'
+
+export interface BusinessSnapshotIF {
+  businessData: {
+    business: BusinessInformationIF
+  },
+  businessAliases: {
+    aliases: Array<NameTranslationSnapshotIF>
+  },
+  businessAddresses: IncorporationAddressIf,
+  contactInfo: BusinessContactIF,
+  businessDirectors: {
+    directors: OrgPersonIF
```

Comment:
Why nest this?

Reply:
Or "businessData" and "businessAliases" above?

Reply:
This was the way the data is returned from the backend and also due to creating on a single request method for the varying datatypes. I will look at refactoring it, to de-structure prior to this point.

Reply:
Fixed in next commit!